forgit.plugin.fish

```diff
@@ -136,14 +137,16 @@ function forgit::reset::head
     set files (git diff --cached --name-only --relative | env FZF_DEFAULT_OPTS="$opts" fzf --preview="$cmd")
     if test -n "$files"
         for file in $files
-            echo $file | tr '\n' '\0' |xargs -I{} -0 git reset -q HEAD {} && git status --short && return
+            echo $file | tr '\n' '\0' |xargs -I{} -0 git reset -q HEAD {}
         end
+        git status --short
+        return
     end
     echo 'Nothing to unstage.'
 end
 
 # git checkout-restore selector
-function forgit::restore
+function forgit::checkout_file
```

Comment:
@wfxr How would you feel about changing all the "restore" variables to "checkout_file" just to match the alias?

for example, these functions would be renamed, but then we would also rename $forgit_restore, and change the comment to say "nothing to checkout". 

I think it may be a bit clearer as to what's going on!